docfiles/docs.js

```diff
@@ -257,8 +258,19 @@ function renderSnippets() {
     });
 }
 
+function responsiveResize(){
+    // Add Responsive Adjustments depending on screen size 
+    $( window ).resize(function() {
```

Comment:
got it. Will refactor this to use pure vanilla, or are we planning to use something different in docs? This will also have to do with issue https://github.com/Microsoft/pxt/issues/5347 regarding lang picker. We can always just use vanilla js :)

Reply:
I was checking that Semantic UI depends on jquery 😟

I found this https://github.com/morganbarrett/Vanilla-Semantic-UI that tries to remove that dependency. 

I think in this case (docs) we would need to talk more about this. What do you think?

Reply:
Perfect, I'll try to not use jquery if possible 😄